examples/blog-starter/package.json

```diff
@@ -7,7 +7,6 @@
     "now-build": "next build && yarn build:rss"
   },
   "dependencies": {
```

Comment:
Upgrade Next version please.

Reply:
This one actually is ok without, it never used .css 😄